tests/func/run/subnet.yml

```diff
@@ -0,0 +1,21 @@
+# FIXME: Find a way we can test the whole playbook rather than just
```

Comment:
Not sure how much you've looked through the func test structure yet, so just a heads up: this `run` file will need `seed` and `cleanup` counterparts.